### ISSUE TITLE
Flutter app stops in android 8

### ISSUE THREAD

**Issue by lucasSzavara** [flutter/flutter#30732]
## Steps to Reproduce

1. I have a screen with a ListView
2. I scrolled to the bottom of the page
3. I try to scroll back to top

when I tried to scroll back, it just stop, doesn't print nothing in the output.
Flutter analyze and flutter doctor doesn't point anything
in others android versions, it works just fine

Flutter doctor:
```
[✓] Flutter (Channel stable, v1.2.1, on Linux, locale pt_BR.UTF-8)
    • Flutter version 1.2.1 at /home/lucas/flutter
    • Framework revision 8661d8aecd (8 weeks ago), 2019-02-14 19:19:53 -0800
    • Engine revision 3757390fa4
    • Dart version 2.1.2 (build 2.1.2-dev.0.0 0a7dcf17eb)

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /home/lucas/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-Q, build-tools 28.0.3
    • Java binary at: /home/lucas/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[✓] Android Studio (version 3.3)
    • Android Studio at /home/lucas/android-studio
    • Flutter plugin version 33.4.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] VS Code (version 1.33.0)
    • VS Code at /usr/share/code
    • Flutter extension version 2.25.0

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.0.0 (API 26)
      (emulator)

• No issues found!
```
I already tried flutter clean, but that didn't changed anything. that happens in emulators and physical devices and I am using linux mint 19.1 with kernel 4.15.0-20-generic

**Comment by escamoteur** [comment]
Hi, sorry for the delay, I just want to check if you could solve this problem.

**Comment by lucasSzavara** [comment]
Hi, I solved that updating flutter to 1.7.8+hotfix.4, sorry for not closing this
